runtime/vm/KeyHashTable.c

```diff
@@ -543,6 +558,13 @@ hashClassTableNextDo(J9HashTableState *walkState)
 	return (NULL == next) ? NULL : next->ramClass;
 }
 
+static BOOLEAN
+isMHProxyPackage(J9ROMClass *romClass) {
+	const char* mhproxy = "jdk/MHProxy";
+	const char* packageName = (const char*)J9UTF8_DATA(J9ROMCLASS_CLASSNAME(romClass));
+	return J9ROMCLASS_IS_HIDDEN(romClass) && 0 == strncmp(packageName, mhproxy, strlen(mhproxy));
```

Comment:
Not all hidden classes can be individually unloaded. Hidden class that can be individually unloaded has flag J9AccClassAnonClass set. So instead of using `J9ROMCLASS_IS_HIDDEN()`, it is more accurate to check flag J9AccClassAnonClass.